Cargo.toml

```diff
@@ -21,7 +21,7 @@ path = "src/bin/wasmtime.rs"
 doc = false
 
 [dependencies]
-wasmtime = { path = "crates/wasmtime", version = "0.33.0", default-features = false, features = ['cache', 'cranelift'] }
+wasmtime = { path = "crates/wasmtime", version = "0.33.0", default-features = false, features = ['cache', 'cranelift', 'pooling-allocator', 'memfd'] }
```

Comment:
Ah, these were necessary to actually see commandline `wasmtime` use memfd I think -- the `default-features = false` otherwise turns everything off. (If there's a better way to arrange this then let me know -- I'm actually not sure I fully grok why `default-features = false` is right for the CLI tool)